### PR TITLE
SUBMARINE-958. Experiment Tag at Customized Experiment

### PR DESCRIPTION
**Pull request by noidname01**
### What is this PR for?

This is the frontend implementation of [SUBMARINE-943](https://issues.apache.org/jira/browse/SUBMARINE-943)
Users can add multiple tags to mark experiments with their own category.
Also add e2e test for tags.

### What type of PR is it?
[Feature]

### Todos

None

### What is the Jira issue?

https://issues.apache.org/jira/projects/SUBMARINE/issues/SUBMARINE-958

### How should this be tested?

Just fill in the tags when creating an experiment.

### Screenshots (if appropriate)

2021-08-06 11-46-03 的螢幕擷圖

https://user-images.githubusercontent.com/55401762/129542663-3a61f277-d0cf-49ee-9147-afc648807c84.mp4

2021-08-16 17-30-42 的螢幕擷圖
### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? No

### CODE REVIEW

File: submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-form/experiment-customized-form/experiment-customized-form.component.ts [hunk 1]
```diff
@@ -133,6 +136,10 @@ export class ExperimentCustomizedFormComponent implements OnInit, OnDestroy {
     });
 
     this.subscriptions.push(sub1, sub2);
+
+    // get tags from server
+    this.listOfOption = []; //TODO
```
Comment (pingsutw) [inline submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-form/experiment-customized-form/experiment-customized-form.component.ts:141]:
```suggestion
    // TODO: get tags from server
    this.listOfOption = [];
```

**Review by pingsutw — COMMENTED** [review]
Other LGTM

File: submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-form/experiment-customized-form/experiment-customized-form.component.ts (hunk 1 above)
Comment (noidname01) [reply submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-form/experiment-customized-form/experiment-customized-form.component.ts:141]: Oh you're right, I'll fix it.

Comment (pingsutw) [reply submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-form/experiment-customized-form/experiment-customized-form.component.ts]: Thanks~

**Comment by noidname01** [comment]
Done~

**Comment by pingsutw** [comment]
Hi @noidname01, Could you fix the conflict? Thanks.

**Comment by noidname01** [comment]
> Hi @noidname01, Could you fix the conflict? Thanks.

@pingsutw Fixed~

**Comment by pingsutw** [comment]
@noidname01 sorry, there is another conflict in the PR, could you fix it?
I will merge it first. Thanks~

**Comment by noidname01** [comment]
@pingsutw I also add e2e test for tags.

**Pull request closed without merging**